velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -1524,6 +1524,94 @@ PlanBuilder& PlanBuilder::window(
       sortingOrders,
       windowNames,
       windowNodeFunctions,
+      false,
+      planNode_);
+  return *this;
+}
+
+PlanBuilder& PlanBuilder::streamingWindow(
+    const std::vector<std::string>& windowFunctions) {
+  VELOX_CHECK_GT(
+      windowFunctions.size(),
+      0,
+      "Window Node requires at least one window function.");
+
+  std::vector<core::FieldAccessTypedExprPtr> partitionKeys;
```

Comment:
Is this a copy-paste? Please, refactor if so.

Please, add checks to verify that all partition by and sort by clause are the same.

Reply:
@mbasmanova [here](https://github.com/facebookincubator/velox/blob/4b62ffdb650f7c3a2bb3df57ef740ebe34858677/velox/exec/tests/utils/PlanBuilder.cpp#L1478) already checks all the partition by and sort by are same.